scripts/e2e.sh

```diff
@@ -25,7 +25,7 @@ echo "e2e_tower_username: '"$TOWER_USERNAME"'" >> e2e_test_config.yml
 echo "Bootstrapping tower in openshift project: ${TOWER_OPENSHIFT_PROJECT}"
 ansible-playbook -i ./inventories/hosts \
 playbooks/bootstrap.yml \
---extra-vars "@e2e_test_config.yml"
+--extra-vars "@e2e_test_config.yml" \
```

Comment:
This added `\` is causing the failure in the e2e test.

Reply:
I'd also recommend a rebase as some changes to the script have been merged which improve the cleanup.